.mergify.yml

```diff
@@ -45,12 +42,3 @@ pull_request_rules:
       backport:
         branches:
           - release/v9.0.x
-
-  - name: Backport patches to the release/v10.0.x branch
```

Comment:
Why remove 10 if we'll need to add it back soon?

Reply:
Good point. I added it back.